src/components/site/aboutPage/aboutPage.js

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import locationIcon from  '../../../assets/images/placeholder.svg';
+import locationImage from '../../../assets/images/location.svg'
+import Styled from '@emotion/styled';
+import { NavLink} from 'react-router-dom';
+
+
+const H1= Styled.h1`
+text-align: center;
+margin-left: 5%;
+margin-top: 10px;
+font-family:'Ubuntu', sans-serif;
+`;
+const P=Styled.p`
+text-align: center;
+width: 60%;
+height:35%;
+font-family:'Ubuntu', sans-serif;
+margin-bottom: 2%
+
+`;
+const Pa=Styled.p`
+text-align: left;
+`;
+const Div= Styled.div`
+display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    flex-direction: column;
+    height: 78vh;
+    width: 50%;
+    flex-basis: 4rem;
+    bottom: 10px;
+    margin-left: 27%;
+`;
+const H2=Styled.h2`
+text-align: center;
+margin-bottom: 12px;
+`;
+const Img= Styled.img`
+width:50px;
+height:20px;
+`;
+const DivlocationIcon = Styled.div`
+margin-bottom: 0.5%;
+`;
+const Footer = Styled.footer`
+display: flex;
+    justify-content: center;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    background-color: rgb(46, 125, 245);
+    color: white;
+    text-align: center;
+    padding: 8px;
+    font-family:'Ubuntu', sans-serif;;
+`;
+const Image=Styled.img`
+border:solid;
+height: 200px;
+margin-top: 0%;
+`;
+const A=Styled.a`
+text-decoration: none;
+color: white;
+`;
+const Navbar=Styled.div`
+box-shadow: 0 1px 6px 0 rgba(32, 33, 36, 0.28);
+padding: 15px 10px 10px 10px;
+background-color: rgb(12, 104, 243);
+display: flex;
+justify-content: center;
+align-items: center;
+margin-bottom: 0%;
+`;
+const Ul =Styled.ul`
+    display: flex;
+    margin-left: 700px;
+    margin-right: 100px;
+`;
+const Li =Styled.li`
+    padding-left: 8px;
+    list-style: none;
+    margin-left: 10px;
+`;
+const Span=Styled.span`
+    color: #fff;
+    font-size: 17px;
+    text-decoration: none;
+    font-family:'Ubuntu', sans-serif;
+`;
+
+const aboutPage = ()=>{
+  
+  return(
+    <div>
+      <Navbar>
```

Comment:
The Navbar should be refactored to its own component should that it can be shared across multiple pages.

Reply:
I have changed it